### PR TITLE
Alignment Updates and Exemption Review Unit Tests

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17420

*Description of changes:*
* Alignment updates for MH Registrations
* Minor style updates
* Clean up instances of global style class 'selected-radio' where not applicable
* Unit tests for Exemptions Review

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1586-8vgcs4br.web.app

**Review by doug-lovett — APPROVED** [review]
LGTM.

File: ppr-ui/tests/unit/ExemptionReview.spec.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  })
+
+  it('renders the FolioOrReferenceNumber for Qualified Supplier', async () => {
+    setupMockQualifiedSupplier()
+    await nextTick()
+
+    expect(wrapper.findComponent(FolioOrReferenceNumber).exists()).toBe(true)
+  })
+
+  it('renders the ConfirmCompletion section', () => {
```
Comment (dimak1) [inline ppr-ui/tests/unit/ExemptionReview.spec.ts:64]: What's the reason for having them in separate tests? So we can add more assertions as we go on?

File: ppr-ui/tests/unit/utils/helper-functions.ts [hunk 2]
```diff
@@ -120,3 +130,9 @@ export function setupMockStaffUser (): void {
   setupMockUser()
   store.setAuthRoles(['staff', 'ppr_staff'])
 }
+
+export function setupMockQualifiedSupplier (): void {
+  setupMockUser()
+  store.setAuthRoles([])
```
Comment (dimak1) [inline ppr-ui/tests/unit/utils/helper-functions.ts:136]: Probably need to specify that this is Lawyer QS.
In my QS tests, I did add `[AuthRoles.MHR_TRANSFER_SALE]` roles, with a `[ProductCode.MANUFACTURER]` product codes.

**Review by dimak1 — APPROVED** [review]
Nice!

File: ppr-ui/tests/unit/ExemptionReview.spec.ts (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/tests/unit/ExemptionReview.spec.ts:64]: Yea, it's just long-hand i guess. Keep it simple.

File: ppr-ui/tests/unit/utils/helper-functions.ts (hunk 2 above)
Comment (cameron-eyds) [reply ppr-ui/tests/unit/utils/helper-functions.ts:136]: Sure, can update. 
Although the Roles no longer have a part to play, for the UI at least. We should just be relying on the product code.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1586-8vgcs4br.web.app

**Pull request merged**